code/cavity2d/Makefile

```diff
@@ -0,0 +1,12 @@
+# Default Target
+.PHONY: all
+all: CAVITY_2D_STAGGERED
+
+# CAVITY_2D_STAGGERED
+CAVITY_2D_STAGGERED: clean
+	gfortran -o CAVITY_2D_STAGGERED CAVITY_2D_STAGGERED.f90
```

Comment:
No optimization flag is specified.  Why?

Reply:
change to gfortran -O3 CAVITY_2D_STAGGERED.f90

Reply:
I am not seeing update?